src/views/ShipmentDetails.vue

```diff
@@ -228,4 +231,13 @@ img {
 .product-count {
   min-width: 9ch;
 }
+
+@media (min-width: 636px) {
+  .shipment-scanner {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
can we do this without a media query if we use auto fit/fill properties of css grid

Reply:
Used auto-fit property